cognite/neat/_rules/transformers/_converters.py

```diff
@@ -658,7 +658,7 @@ def as_dms_rules(
         info_metadata = self.rules.metadata
         default_version = info_metadata.version
         default_space = self._to_space(info_metadata.prefix)
-        metadata = self._convert_metadata_to_dms(info_metadata)
+        self.dms_metadata = self._convert_metadata_to_dms(info_metadata)
```

Comment:
```suggestion
        dms_metadata = self._convert_metadata_to_dms(info_metadata)
```
No need to make it attach to the class.